test/test.getch.ts

```diff
@@ -604,25 +601,56 @@ describe('🥁 configuration options', () => {
   });
 
   it('should not stringify the data if it is appended by a form', async () => {
+    const FormData = (await import('node-fetch')).FormData;
```

Comment:
With the move to Node 18 do we have to use `FormData` from `node-fetch` or can we use the internal Node version?

Reply:
Caveat the stability level is still `experimental` in Node 18.

Reply:
It doesn't appear node-fetch supports the native `FormData`, unfortunately. If one is using native `fetch` (via `fetchImplementation`) they can use the native `FormData`.